.changeset/fair-pets-crash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/vite-plugin-svelte': major
+---
+
+Remove experimental "advanced raw queries" feature. Basic `File.svelte?raw` is still supported.
```

Comment:
Does this mean we should remove vite_css_regex from https://github.com/sveltejs/kit/blob/fb88ff7f2eeeec850130808e79e40191f9b273cb/packages/kit/src/exports/vite/dev/index.js#L247 ?